src/onegov/org/forms/event.py

```diff
@@ -642,6 +642,8 @@ class EventConfigurationForm(Form):
             ValidFilterFormDefinition(
                 require_email_field=False,
                 require_title_fields=False,
+                reserved_fields={name for name, _ in
+                                 EventForm()._unbound_fields}
```

Comment:
```suggestion
                                 EventForm._unbound_fields}
```
You don't need to create an instance, `_unbound_fields` is a class attribute.

Reply:
You could also use `onegov.form.utils.get_fields_from_class` which works even if no instance has been created yet.

Reply:
Better, then creating an instance!